DEV-10/Inputer.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace CollectOneArrayFromN
+{
+  // Class contains method
+  // that returns valid numbers entered by user
+  public class Inputer
+  {
+    private const string BadValue = "Bad value. Try again.";
+    private const string IntValue = "Please, enter N - count of arrays: ";
+
+    // Method returns positive integer number entered by user from console
+    public int ReturnPositiveInt()
```

Comment:
Bad name, single responsibility. This method not only return positive int, read it from console